modules/end-user-guide/partials/proc_building-custom-intellij-editor-images.adoc

```diff
@@ -0,0 +1,154 @@
+[id="building-custom-intellij-editor-images_{context}"]
+= Building custom IntelliJ editor images
+
+This procedure describes how to build an IntelliJ-based editor image using a custom archive and a custom image name.
```

Comment:
`-based`?

Reply:
Replaced by: This procedure describes how to build an image for an IDE based on the IntelliJ Platform up to version 2020.3. See the list of compatible IDEs.